Pipe 14601 (#6853)
harness/developer-hub: release-notes/continuous-delivery.md

#### Behavior change

- There is a change in the permissions associated with [Overrides V2](/docs/continuous-delivery/x-platform-cd-features/overrides-v2):
  - Previously: for service-specific, service, and infrastructure-specific overrides, the environment Create and Edit permissions were required.
  - Now: the environment Create and Edit permissions are not required. Only the service Create and Edit permissions are required.
  - Here is the comprehensive list of permissions required across different types:
    - Service and infrastructure-specific: New, edit, clone, and delete actions:
      - Service permission: `Create/Edit`
    - Infrastructure-specific: New, edit, clone, and delete actions:
      - Environment permission: `Create/Edit`
    - Service-specific:  New, edit, clone, and delete actions:
      - Service permission: `Create/Edit`
    - Global Environment: New, edit, clone, and delete actions:
      - Environment permission: `Create/Edit`

- This is a change in behavior of Rerun execution, when a Matrix is involved. (PIPE-14601)

>   When you select **Rerun from Last Failed Stage**, the flow remains the same but you can choose from a failed stage and the run begins from there. Earlier, the last failed stage ran by default. If there are failed stages in a Matrix/Loop, the execution begins from only the failed stages (skipping the successful ones), and moves on to any others that haven’t run yet.

Reviewer: Perhaps we don't need the "but you can choose from a failed stage..."? There's no choice in this case, the run automatically starts from the last failed stages. 

Otherwise, this looks good.

>   When you select **Rerun from Last Failed Stage**, the flow remains the same but you can choose from a failed stage and the run begins from there. Earlier, the last failed stage ran by default. If there are failed stages in a Matrix/Loop, the execution begins from only the failed stages (skipping the successful ones), and moves on to any others that haven’t run yet.

Author: ```suggestion
  When you select **Rerun from Last Failed Stage**, the run automatically starts from the last failed stage. If there are failed stages in a Matrix/Loop, the execution begins from only the failed stages (skipping the successful ones), and moves on to any others that haven’t run yet.
```
